i18n.csv

```diff
@@ -10,14 +10,14 @@ ethError,"The address doesn't seem right. Any chance you put in your BTC address
 faq,"https://github.com/HyperLootProtocol/Starter-Pack/wiki/Bounty-FAQ"
 lvlUp, "Hey, congratulations, <@${id}>! Level up! Your new level is ${lvl}, keep it up!"
 support,"You can get all of the questions answered in the Discord chat here: https://discord.gg/qEH8PHk, but first check out the FAQ page https://github.com/HyperLootProtocol/Starter-Pack/wiki/Bounty-FAQ"
-startHey,"Hey, ${username}"
+startHey,"Hey, <@${id}>"
 start,"My name is Hyper and I'm an expert on all things Hyperloot!💰 It's dangerous to go alone, so take a list of missions you can do to get your FREE HLT tokens:"
 terms,"https://github.com/HyperLootProtocol/Starter-Pack/wiki/Bounty-Terms-&-Conditions"
 help,"${PREFIX}${command} - ${help}"
 guildMemberAdd,"<@${id}>, welcome to our server!"
 status,"Ayo!\nLevel: ${curLvl}\nExp: ${exp}\nNext level on: ${expToNext}"
 statusError,"Something's wrong with your status, mate =\\"
-,
+,s
```

Comment:
Oo what it??

Reply:
ouch... missclick ;D

Reply:
to prevent this, try use `git add -p` (interactive mode) and check all what u add to commit =)